tests/raftstore/pd.rs

```diff
@@ -151,45 +141,215 @@ impl Cluster {
     fn get_stores(&self) -> Vec<metapb::Store> {
         self.stores.values().map(|s| s.store.clone()).collect()
     }
+
+    fn handle_heartbeat_version(&mut self, region: metapb::Region) -> Result<()> {
+        // For split, we should handle heartbeat carefully.
+        // E.g, for region 1 [a, c) -> 1 [a, b) + 2 [b, c).
+        // after split, region 1 and 2 will do heartbeat independently.
+        let end_key = enc_end_key(&region);
+        let start_key = enc_start_key(&region);
+        assert!(end_key > start_key);
+
+        let version = region.get_region_epoch().get_version();
```

Comment:
I  don't think rv is a good name, wha't rv for? region_version? If so, region_version is better, but here I think version is ok, because we just handle region epoch version and conf version here.